pkg/kubectl/cmd/drain_test.go

```diff
@@ -32,12 +32,13 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
-
+	"k8s.io/api/core/v1"
```

Comment:
Thanks for pointing me to the doc above. Now I understand the problem. But AFAIU, the second point that I made still holds good.(i.e unit tests need to be written against v1, if we have created v1 object in code).

Also, if you can bump the bot again for running unit tests and give lgtm, it'd be great. I have couple of more patches lined up based on these changes.